package.json

```diff
@@ -17,8 +17,14 @@
     "react-navigation": "^1.0.0-beta.11"
   },
   "devDependencies": {
+    "babel-eslint": "^7.2.3",
     "babel-jest": "20.0.3",
     "babel-preset-react-native": "2.0.0",
+    "eslint": "^3.19.0",
+    "eslint-config-airbnb": "^15.0.1",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^5.0.1",
```

Comment:
For those who wonder what this `a11y` is, "The A11Y Project. A community-driven effort to make web accessibility easier."( http://a11yproject.com/ )